### PR TITLE
Mandatory packaging of output files

### PR DESCRIPTION
**Pull request by DunklesArchipel**
Checklist
* [x] Added an entry in `doc/news/`. 
* [x] Added a test for this change.
* [x] Updated the documentation. (will be addressed separately #177 )

Dependencies:
- [x] #178

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,9 @@ Usage: svgdigitizer [OPTIONS] COMMAND [ARGS]...
 Options:
   --help  Show this message and exit.
 Commands:
-  cv        Digitize a cylic voltammogram.
+  cv        Digitize a cylic voltammogram and create a frictionless...
   digitize  Digitize a plot.
+  figure    Digitize a figure with units on the axis and create a...
```
Comment (saraedum) [inline README.md:31]: Why the `...`?

File: doc/news/packaging.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+**Added:**
+
+* Added CLI command `svgdigitzer figure` which allows creating a datapackage from a figure with units on the axis.
+
+**Changed:**
+
+* Refactored `entrypoint.py`.
```
Comment (saraedum) [inline doc/news/packaging.rst:7]: I don't think this is a "Changed". I would just remove this section.

File: svgdigitizer/entrypoint.py [hunk 3]
```diff
@@ -10,8 +10,9 @@
     Options:
       --help  Show this message and exit.
     Commands:
-      cv        Digitize a cylic voltammogram.
+      cv        Digitize a cylic voltammogram and create a frictionless...
       digitize  Digitize a plot.
+      figure    Digitize a figure with units on the axis and create a...
```
Comment (saraedum) [inline svgdigitizer/entrypoint.py:15]: Why the `...`?

File: svgdigitizer/entrypoint.py [hunk 4]
```diff
@@ -188,29 +188,93 @@ def digitize(svg, sampling_interval, outdir, skewed):
...
-    help="sampling interval on the x-axis in volts",
-)
+@click.command(name="figure")
+@sampling_interval_option
+@outdir_option
 @click.option(
     "--metadata", type=click.File("rb"), default=None, help="yaml file with metadata"
 )
-@click.option("--package", is_flag=True, help="create .json in data package format")
+# @click.option("--package", is_flag=True, help="create .json in data package format")
```
Comment (saraedum) [inline svgdigitizer/entrypoint.py:197]:
```suggestion
```

File: svgdigitizer/entrypoint.py [hunk 5]
```diff
@@ -188,29 +188,93 @@ def digitize(svg, sampling_interval, outdir, skewed):
...
+@sampling_interval_option
+@outdir_option
 @click.option(
-    "--outdir",
-    type=click.Path(file_okay=False),
-    default=None,
-    help="write output files to this directory",
+    "--metadata", type=click.File("rb"), default=None, help="yaml file with metadata"
 )
+# @click.option("--package", is_flag=True, help="create .json in data package format")
```
Comment (saraedum) [inline svgdigitizer/entrypoint.py:271]:
```suggestion
```

**Review by saraedum — APPROVED** [review]

File: svgdigitizer/entrypoint.py (hunk 3 above)
Comment (DunklesArchipel) [reply svgdigitizer/entrypoint.py:15]: doctest creates a cutoff at 80 characters and the line above is the output from doctest.

Somehow `+NORMALIZED_WHITESPACE` does not work in this case.

File: README.md (hunk 1 above)
Comment (DunklesArchipel) [reply README.md:31]: Same as below. `byexample` creates a cut-off at 80 characters.

File: svgdigitizer/entrypoint.py [hunk 6]
```diff
@@ -306,16 +363,26 @@ def digitize_cv(
...
             metadata=metadata,
             force_si_units=si_units,
         )
 
+    _create_outfiles(
+        svgfigure=svgfigure, svg=svg, outdir=outdir, bibliography=bibliography
+    )
+
+
+def _create_outfiles(svgfigure, svg, outdir, bibliography):
```
Comment (DunklesArchipel) [inline svgdigitizer/entrypoint.py:377]: @saraedum I added a new helper function, to reduce duplicate code in the different digitization commands in the CLI. I guess this is fine.

File: README.md (hunk 1 above)
Comment (DunklesArchipel) [reply README.md:31]: see #187

File: svgdigitizer/entrypoint.py (hunk 3 above)
Comment (DunklesArchipel) [reply svgdigitizer/entrypoint.py:15]: see #187

**Pull request merged**